### DISCUSSION TITLE
Creating Lighting Demo build files

### DISCUSSION THREAD

**Discussion by luna215** [molstar/molstar#539] [Q&A] [answered]
How would I go about creating the lightning demo build files? I see that we have the demo .ts file under `/Users/paulluna/development/molstar-private/src/examples/lighting/index.ts`, so how would I go about creating the `index.html` file?

**Comment by dsehnal** [comment, marked as answer]
``npm run build`` and then go to the ``build/examples`` folder in the root dir. ``npm run watch`` will do the same thing with rebuild on changes.
